addon/TabTracker.js

```diff
@@ -177,7 +178,7 @@ TabTracker.prototype = {
       return;
     }
     if (this._tabData.start_time) {
-      this._tabData.session_duration = (Date.now() - this._tabData.start_time);
+      this._tabData.session_duration = (absPerf.now() - this._tabData.start_time);
```

Comment:
So I just did the drop-in replacement of Date.now() for the moment.  We can do further refinement once we have a caller for higher resolution timing.